app/src/main/java/a/a/a/Dp.java

```diff
@@ -169,7 +168,24 @@ public void a(iI iIVar, String str) {
      * @throws Exception Thrown if dexing had problems
      */
     public final void a(String outputPath, ArrayList<String> dexes) throws Exception {
-        dexLibraries(outputPath, dexes);
+        try {
+            if (!Integer.parseInt(this.a) >= 21) {
```

Comment:
![image](https://user-images.githubusercontent.com/74397286/142867660-3d4fd64e-4a4a-4d54-af5b-0f0281475b34.png)
! cant be directly applied sir
surround when like picture

Reply:
or just invert the operator